easysocial/easysocial/profile.php

```diff
@@ -47,7 +47,12 @@ function getProfile()
 		$data = array();
 		
 		$user = FD::user($userid);
-		
+		/* Check whether user is not deleted */
+		if($user->username == null){
```

Comment:
Please change this to user id for semantic reasons. Should be simply `if ($user->id)`